### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by rsnazario**
This Milestone is Designed to:

- [x] Remind that no need to time-wasting on styling so far.

------------------------------------- milestone on website -------------
Milestone 3
[Solo Project]
Your components are set and ready to work, but they don’t look really pretty, do they?

This milestone is here only to remind you that you should not do anything about styling for this project at this moment.

Do not worry though! You will be asked to make it pretty very soon!

-------------------------------------------------------------------------------

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -10,11 +10,13 @@ export default class App extends React.Component {
 
   render() {
     return (
-      <div className="app-panel">
-        <h1>Hello from App</h1>
-        <Display />
-        <ButtonPanel />
-      </div>
+      <>
+        <div className="app-panel">
```
Comment (christianotieno) [inline src/components/App.js:14]: ## Changes:

#### ADD Flex display mode for these components and other missing features for this milestone.

- [x] Please add flex display mode for this component, the `Display` and `ButtonPanel` components ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png))

- [x] The calculator width needs to strickly equal 700px as instructed in the requirements ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/700Wide.png)).
 ##

#### `Display` component

* [x] As instructed in this milestone, this component needed to have a grey background. Let's implement this. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-bg.png)). 
* [x] It needs also, to strickly have a height of 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-height.png)).
* [x] While at it, remember to also add a reasonable padding from the edges ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-padding.png)).
##
#### `ButtonPanel` component
* [x] Ensure that each row's height is 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png)).
##

#### `Button` component
- For this component: 👇 
* [x] Ensure that buttons take 25% of container width (except `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/25Wide.png)).
* [x] Ensure that buttons have borders ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/ButtonsBoarder.png)).
* [x] Ensure that buttons text is centered and black ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/TextBlack.png)).
* [x] Ensure that it accepts `color` and `wide` as an incoming props ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png)).
* [x] Ensure that the default `color` for button is orange ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)).
* [x] Ensure that `wide` is boolean, true means that button take 50% of space (used for `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png)).
##
### Please note:
**These changes are mandatory only if you are not using the `v1.1` of React, [Milestone description here](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106475924/).**
##

**Review by christianotieno — CHANGES_REQUESTED** [review]
### Changes Required ♻️ 
Hello @rsnazario 👋 

Congratulations on the great work done on this milestone. The project is coming up just fine. I love how you are modularising your code and decluttering everything. Keep it up.

We do have some changes to work on before we proceed. Take a look below

### Happy coding 💻 
#### Christian Otieno (TSE)

File: src/components/App.js (hunk 1 above)
Comment (rsnazario) [reply src/components/App.js:14]: Hello man, although I agree with most of the changes required, the point of this whole Milestone is just a reminder to not waste any time styling it so far. All the requirements you have provided me now are for the final version of the project. 
At the edited description for the pull request, you can read the description from the curriculum.

**Review by lucasmazo23 — APPROVED** [review]
## Project approved 🚀
### Great job!
### Your project looks great!
### You can go ahead and merge this pull request!
### Happy coding! 💻
## Lucas Mazo

File: src/components/App.js (hunk 1 above)
Comment (christianotieno) [reply src/components/App.js:14]: Hello, @rsnazario Apologies for any confusion this review must have caused you. I understand you are using v1.1 and this milestone ought to be approved without the styling bit. 

I wish you all the very best in your next tasks. Cheers!

### Happy coding 💻 

#### Christian Otieno (TSE)

Comment (rsnazario) [reply src/components/App.js:14]: No problem man, I understand that you were aiming for my best result and I appreciate it :) 
It also helps for the next milestones hahaha
I hope you have a great weekend

**Pull request merged**